reference-architecture/rhv-ansible/ocp-vars.yaml

```diff
@@ -54,6 +55,25 @@ load_balancer_hostname: openshift-lb.example.com
 openshift_master_cluster_hostname: openshift-lb.example.com
 openshift_master_cluster_public_hostname: openshift.example.com
 
+# OpenShift Identity Providers
+# htpasswd shown here, other options documented at
+# https://docs.openshift.com/container-platform/3.5/install_config/configuring_authentication.html
+openshift_master_identity_providers:
+  - name: htpasswd_auth
+    login: true
+    challenge: true
+    kind: HTPasswdPasswordIdentityProvider
+    filename: /etc/origin/master/htpasswd
+# Defining htpasswd users
+#openshift_master_htpasswd_users:
+#  - user1: <pre-hashed password>
+#  - user2: <pre-hashed password>
+# Example with admin:changeme
```

Comment:
@e-minguez Good idea, though upstream didn't bother including instructions :-) That can go in the RefArch, yes?